glossary.md

```diff
@@ -0,0 +1,23 @@
+# Glossary
+To avoid confusion, see this list of frequently used terms.
```

Comment:
Hm, I feel like we should not have to explain what a glossary _is_, so maybe even better to just drop this sentence?

Reply:
I stumbled upon the glossary, because at first it seemed to repeat the same information from the file I read before and I wondered what the difference is. Probably a problem raised by the PR, not for usage afterwards.